Makefile

```diff
@@ -104,6 +105,8 @@ src/js/pyproxy.gen.ts : src/core/pyproxy.* src/core/*.h
 build/test.html: src/templates/test.html
 	cp $< $@
 
+build/module_test.html: src/templates/module_test.html
```

Comment:
At some point we probably should try to combine some of these templates build rules (no action needed in this PR).